README

```diff
@@ -1,7 +1,7 @@
 This is the MacPorts documentation directory.
 
 To generate it you have to checkout the entire "macports-guide" repository,
-and you need the "docbook-xsl", "docbook-xml-4.5" and "libxslt" ports.
+and you need the "docbook-xsl", "docbook-xml-5.1" and "libxslt" ports.
```

Comment:
You're referencing the docbook-xml-5.1 port here.

Reply:
fixed. I think y'all figured out my original intent to go to 5.1 before realizing that would be harder :)